src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/UIPropertyEditorDataProducer.cs

```diff
@@ -64,6 +64,18 @@ IEnumerable<IEntityValue> createEditorValues()
                     IEntityValue editorValue = CreateEditorValue(parent, editor, properties);
                     yield return editorValue;
                 }
+
+                if (property is StringProperty stringProperty)
+                {
+                    if (stringProperty.Subtype == "file")
+                    {
+                        yield return CreateEditorValue(parent, new ValueEditor { EditorType = "FileBrowse" }, properties);
```

Comment:
Can we say "FileBrowse**r**" instead of "FileBrowse", as in the editor we want is a file browser?

Reply:
The other editor types are things like "String", "Bool", "Enum". Does that change anything for you?

Reply:
Maybe "Path" makes more sense.

Reply:
Makes sense to me.